### PR TITLE
Fixed styling issue

### PR DESCRIPTION
**Pull request by ruks**
## Purpose
Fixed styling issue

### CODE REVIEW

File: components/org.wso2.analytics.apim.widgets/AppAndAPIErrorTable/src/AppAndAPIErrorTablewidget.jsx [hunk 1]
```diff
@@ -742,78 +742,78 @@ class AppAndAPIErrorTablewidget extends Widget {
                                     defaultMessage='Error Summary'
                                 />
                             </h3>
+                            <FormControl component='fieldset'>
```
Comment (ruwiniwj) [inline components/org.wso2.analytics.apim.widgets/AppAndAPIErrorTable/src/AppAndAPIErrorTablewidget.jsx:745]: why are we adding the content inside the heading wrapper div? that div is dedicated for the header styling only right? don't we need to have content in a different div and add the required styling for the content in that div.

File: components/org.wso2.analytics.apim.widgets/APILatencyOverTime/src/APILatencyOverTimeWidget.jsx [hunk 2]
```diff
@@ -138,7 +134,7 @@ class APILatencyOverTimeWidget extends Widget {
             },
             root: {
                 backgroundColor: this.props.muiTheme.name === 'light' ? '#fff' : '#0e1e34',
-                height: '100%',
```
Comment (ruwiniwj) [inline components/org.wso2.analytics.apim.widgets/APILatencyOverTime/src/APILatencyOverTimeWidget.jsx:141]: wouldn't removing the height result in the background shrinking when there is no content?

File: components/org.wso2.analytics.apim.widgets/AppAndAPIErrorTable/src/AppAndAPIErrorTablewidget.jsx (hunk 1 above)
Comment (ruks) [reply components/org.wso2.analytics.apim.widgets/AppAndAPIErrorTable/src/AppAndAPIErrorTablewidget.jsx:748]: Fixed

File: components/org.wso2.analytics.apim.widgets/APILatencyOverTime/src/APILatencyOverTimeWidget.jsx (hunk 2 above)
Comment (ruks) [reply components/org.wso2.analytics.apim.widgets/APILatencyOverTime/src/APILatencyOverTimeWidget.jsx:141]: There was a styling issue due to that. Chart data was not properly formatted due to this.

**Pull request merged**